pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, git
+, nodejs
+, protobuf
+, protoc-gen-go
+, protoc-gen-go-grpc
+}:
+
+buildGoModule rec {
+  pname = "turbo";
+  version = "1.6.3";
+
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "v${version}";
+    sha256 = "csapIeVB0FrLnmtUmLrRe8y54xmK50X30CV476DXEZI=";
+  };
+
+  modRoot = "cli";
+
+  vendorSha256 = "Kx/CLFv23h2TmGe8Jwu+S3QcONfqeHk2fCW1na75c0s=";
+
+  nativeBuildInputs = [
+    git
+    nodejs
```

Comment:
does turbo work with different node versions on runtime?

if no, turbo should be scoped as nodePackages.turbo = nodejs.pkgs.turbo